plugins/infinispan80/pom.xml

```diff
@@ -116,7 +116,7 @@
             </property>
          </activation>
          <properties>
-            <version.infinispan>8.0.1.Final</version.infinispan>
+            <version.infinispan>8.1.0.Beta1</version.infinispan>
```

Comment:
I've created infinispan81 plugin in https://github.com/radargun/radargun/pull/317 to keep isfinispan80 plugin on 8.0.x version line.